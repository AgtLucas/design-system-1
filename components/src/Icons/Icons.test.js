import * as Icons from './';

describe('Icons', () => {
  it('matches snapshot', () => {
    expect(Icons).toBeDefined();
    expect(Icons).toMatchSnapshot();
  });
});

Object.keys(Icons).forEach(name => (
  describe(name, () => {
    it('matches snapshot', () => {
      expect(Icons[name]).toBeDefined();
      expect(Icons[name]).toMatchSnapshot();
    });
  })
));