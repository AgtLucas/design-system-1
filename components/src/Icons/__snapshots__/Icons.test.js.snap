// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons matches snapshot 1`] = `
Object {
  "Icons": Object {
    "Icon": [Function],
    "Svgs": Object {
      "add": [Function],
      "cancel": [Function],
      "check": [Function],
      "company": [Function],
      "delete": [Function],
      "edit": [Function],
      "lock": [Function],
      "menu": [Function],
      "save": [Function],
      "search": [Function],
      "site": [Function],
      "unlock": [Function],
      "user": [Function],
    },
    "names": Array [
      "add",
      "cancel",
      "check",
      "company",
      "delete",
      "edit",
      "lock",
      "menu",
      "save",
      "search",
      "site",
      "unlock",
      "user",
    ],
  },
}
`;

exports[`Icons matches snapshot 2`] = `
Object {
  "Icon": [Function],
  "Svgs": Object {
    "add": [Function],
    "cancel": [Function],
    "check": [Function],
    "company": [Function],
    "delete": [Function],
    "edit": [Function],
    "lock": [Function],
    "menu": [Function],
    "save": [Function],
    "search": [Function],
    "site": [Function],
    "unlock": [Function],
    "user": [Function],
  },
  "names": Array [
    "add",
    "cancel",
    "check",
    "company",
    "delete",
    "edit",
    "lock",
    "menu",
    "save",
    "search",
    "site",
    "unlock",
    "user",
  ],
}
`;
